src/Microsoft.Extensions.Http.Polly/DependencyInjection/PollyServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Polly.Registry;
+
+namespace Microsoft.Extensions.DependencyInjection
+{
+    /// <summary>
+   /// Provides convenience extension methods to register <see cref="IPolicyRegistry{String}"/> and 
+   /// <see cref="IReadOnlyPolicyRegistry{String}"/> in the service collection.
+    /// </summary>
+    public static class PollyServiceCollectionExtensions
```

Comment:
These are just for convenience. Our `AddPolicyHandlerFromRegistry` methods will require you to have registered `IReadOnlyPolicyRegistry<string>` somehow. We don't care if you used these methods to do it.